src/scraping/collector.py

```
@@ -0,0 +1,123 @@
+MVP Data Collection using BrightData
+"""
+from typing import List, Dict, Any
+import asyncio
+import aiohttp
+from ..core.config import (
+    BRIGHTDATA_USERNAME,
+    BRIGHTDATA_PASSWORD,
+    BRIGHTDATA_ZONE,
+    SCRAPING_BATCH_SIZE,
+    SCRAPING_DELAY,
+    MAX_RETRIES
```

Ruff (0.8.2)

`..core.config.MAX_RETRIES` imported but unused

Remove unused import: `..core.config.MAX_RETRIES`

(F401)